executor/standalone/standalone_test.go

```diff
@@ -262,20 +156,22 @@ func dockerPull(t *testing.T, imgName string, imgDigest string) (*dockerTypes.Im
 	return res, err
 }
 
-func testSimpleJob(t *testing.T, jobID string, usePodSpec bool) {
+func TestSimpleJob(t *testing.T) {
+	wrapTestStandalone(t)
 	ji := &JobInput{
 		ImageName:     alpine.name,
 		Version:       alpine.tag,
 		EntrypointOld: "echo Hello Titus",
-		JobID:         jobID,
-		UsePodSpec:    usePodSpec,
+		JobID:         t.Name(),
```

Comment:
What do you think about having a wrapper function here to create the job ID?  So something like:
```
JobID: generateJobID(t.Name)
```

Having a unique job ID (between both the cinfo and pod spec tests) makes it much easier to debug tests when they fail in Buildkite - you can search for the job ID in both the build logs and the journald logs and not have to think too hard about which job you're actually looking at.  This becomes particularly difficult when all of the tests are running in parallel.

Reply:
Oh yea of course. I was too focused on the *test* name, you are right the jobid is a great place to stick in that suffix.

Reply:
Added. We can have our cake (single test with a click) and eat it too (context-aware jobids in the test output)!

Reply:
Wooo, thank-you! 🎉